sql: RSG function tests failing

```
panic: unknown arg type: regtype (parser.tOid)

goroutine 5030174 [running]:
github.com/cockroachdb/cockroach/pkg/internal/rsg.(*RSG).GenerateRandomArg(0xc44a47f980, 0x290e0e0, 0xc420277fa4, 0xc4205f3701, 0x1e)
    /go/src/github.com/cockroachdb/cockroach/pkg/internal/rsg/rsg.go:218 +0x444
github.com/cockroachdb/cockroach/pkg/sql_test.TestRandomSyntaxFunctions.func2(0xc4215ab7c0, 0xc44a47f980, 0xc448aaf6e0, 0xc4643a8900)
    /go/src/github.com/cockroachdb/cockroach/pkg/sql/rsg_test.go:129 +0x18f
github.com/cockroachdb/cockroach/pkg/sql_test.testRandomSyntax.func2()
    /go/src/github.com/cockroachdb/cockroach/pkg/sql/rsg_test.go:233 +0xc0
created by github.com/cockroachdb/cockroach/pkg/sql_test.testRandomSyntax
    /go/src/github.com/cockroachdb/cockroach/pkg/sql/rsg_test.go:244 +0x558
```

@jordanlewis I think you've been doing the oid work? Could you handle this?

--- a/pkg/internal/rsg/rsg.go
+++ b/pkg/internal/rsg/rsg.go
@@ -207,6 +207,11 @@ func (r *RSG) GenerateRandomArg(typ parser.Type) string {
 	case parser.TypeIntArray,
 		parser.TypeStringArray,
 		parser.TypeOid,
+		parser.TypeRegClass,
+		parser.TypeRegNamespace,
+		parser.TypeRegProc,
+		parser.TypeRegProcedure,
+		parser.TypeRegType,
 		parser.TypeAnyArray,
 		parser.TypeAny:
 		v = "NULL"

